tests/driver_at24mac/app.config.test

```diff
@@ -0,0 +1,4 @@
+# this file enables modules defined in Kconfig. Do not use this file for
+# application configuration. This is only needed during migration.
+CONFIG_MODULE_AT24C256=y
```

Comment:
seems unrelated

Reply:
Well, the `at24mac` needs some implementation of `at24cxxx`

Reply:
No it doesn't, only if you want to use the EEPROM functionality.
The ID functionality is separate.

Reply:
I don't understand, here the module adds the dependency:
https://github.com/RIOT-OS/RIOT/blob/a3e4c1eca5bf40215174dcc4e97795d5487fbcbc/drivers/at24mac/Makefile.dep#L1

Also, appears in the pulled modules of the test application:
```
at24cxxx
at24mac
auto_init
auto_init_xtimer
board
core
core_init
core_msg
core_panic
cortexm_common
cortexm_common_periph
cpu
div
newlib
newlib_nano
newlib_syscalls_default
periph
periph_common
periph_gpio
periph_i2c
periph_init
periph_init_gpio
periph_init_i2c
periph_init_pm
periph_init_timer
periph_init_uart
periph_pm
periph_timer
periph_uart
pm_layered
sam0_common_periph
stdio_uart
sys
xtimer
```

Reply:
`at24mac` is *also* an implementation of `at24cxxx`

Reply:
Hmm, ok, will try to model that

Reply:
Done, now this module selects `at24cxxx` as well